nupic/encoders/category.capnp

```diff
@@ -0,0 +1,20 @@
+@0xb639caa166140b0f;
+
+using import "scalar.capnp".ScalarEncoderProto;
+
+# Next ID: 6
+struct CategoryEncoderProto {
+
+  # Next ID: 2
+  struct CategoryMapping {
+    index @0 :UInt32;
+    category @1 :Text;
+  }
+
+  width @0 :UInt32;
+  categoryToIndex @1 :List(CategoryMapping);
+  indexToCategory @2 :List(CategoryMapping);
```

Comment:
Don't store both `categoryToIndex` and `indexToCategory`. When you read it back out it is easy to construct both attributes from just one of these in the proto.